lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -71,4 +57,30 @@ class AppBloc extends Bloc<AppEvent, AppState> {
     _shouldShowCancelGuardianAlertMessage.cancel();
     return super.close();
   }
+
+  void initDynamicLinks() async {
+    FirebaseDynamicLinks.instance.onLink(onError: (error) async {
+      print('onLinkError');
+      print(error.message);
```

Comment:
full page error, snack or just intern debug?

Reply:
debug for now.